### PR TITLE
Adds preset plant gene disks

### PR DESCRIPTION
**Pull request by Pink-Chink**
## About The Pull Request

Adds preset plant gene disks found in a deepmaint vendor and a botany trade ship, the MTV Arbor. Preset disk info TBD.

## Why It's Good For The Game

Outside of a few use cases, botany is a RNG grindfest. While I tried to avoid enabling quick routes to some obvious powergaming plants, this may enable some weird stuff. Taking a "fun now, balance later" approach.

## Changelog
:cl:
add: Added preset plant gene disks
add: Added MTV Arbor, a botany gene disk trade ship
add: Added Eugene's Plant Genes, a deepmaint vendor with powerful plant genes
/:cl:

### CODE REVIEW

**Comment by garrulousTeamotei** [comment]
Teleporting is easy enough to get from other existant plant seeds. Chem spraying, on the other hand, is a maintshroom exclusive, could be a reasonable addition

**Comment by master832** [comment]
Looks interesting. It would be cool if plant analyzer showed biochemistry when analyzing not only fruits, but also seeds.

**Review by SirRichardFrancis — APPROVED** [review]
Guild's mushroom export money printer goes brrr, don't even have to go outside and talk with NT or Moebius.

**Comment by Pink-Chink** [comment]
>Guild's mushroom export money printer goes brrr, don't even have to go outside and talk with NT or Moebius.

They'd have to rob either of em to splice the traits. I expect that this will happen, but I don't think this is going to change Guild's issues either way.

**Comment by SirRichardFrancis** [comment]
Okay, fair. Although I foresee Guild getting "circuit station", or that splicer circuit otherwise becoming widely available in the future. You know, for "vagabond botany" or something.

**Comment by Humonitarian** [comment]
Ghetto splicing would be cool, even if worthless.

Put two plants next to eachother, they grow a third with mixed traits, no disks involved.

File: code/modules/hydroponics/seed_gene_presets.dm [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+	var/list/firstnames = list (
+		"bleak", "bog", "bum", "candy", "coarse", "corpse", "cramp", "club", "demon", "dog", "dung", "felt", "fly", "goblin", "grave", "greasy", "hypoxylon", "jelly", "junk", "icky", "imp", "ling", "lung", "maggot", "monkey", "mottled", "pixie", "poached", "powder", "pterula", "ramularia", "radiant", "rat", "roach", "rock", "scruffy", "serbian", "shaggy", "slimy", "smelly", "smoky", "space", "spider", "spiky", "splash", "stag", "stinky", "tumbling", "undulate", "vagabond", "veiled", "wall", "web"
+		)
+	var/list/secondnames = list(
+		"amanita", "bane", "beacon", "bolete", "bonnet", "bulgar", "cap", "cone", "conocybe", "coral", "clown", "cremini", "crepidotus", "cup", "cushion", "entoloma", "ear", "fungus", "gill", "heart", "horn", "hydnum", "leafspot", "leoninus", "lichen", "micellium", "morsel", "moss", "mushroom", "ori", "oyster", "panellus", "polypore", "panus", "porcini", "porridge", "puffball", "rod", "rot", "russula", "scale", "smut", "spots", "stem", "stool", "tail", "tears", "tongue", "trumpet", "truffle",  "urn", "vase", "wort"
+		)
+
+	var/plant_name = "[pick(firstnames)] [pick(secondnames)]"
+	var/plant_id = rand(100,999)
```
Comment (hyperioo) [inline code/modules/hydroponics/seed_gene_presets.dm:24]: What happens if it randomly ends up twice with the same ID for two drives?

Comment (Pink-Chink) [reply code/modules/hydroponics/seed_gene_presets.dm:24]: Nothing. It's purely for name generation.

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**